rules/windows/builtin/win_mal_service_installs.yml

```diff
@@ -12,25 +14,12 @@ logsource:
 detection:
     selection:
         EventID: 7045
-    malsvc_wce:
```

Comment:
all the sections deleted here due to update of the specific rule — [win_mal_creddumper.yml](https://github.com/Neo23x0/sigma/blob/1f7b3bc9a2d4c1f743da51f4283547d741b3c0e7/rules/windows/builtin/win_mal_creddumper.yml). All the stuff is now there.